hooks/post_gen_project.py

```diff
@@ -72,6 +87,8 @@ def set_up_deployment():
 
     LOG.info('Moving deployment configuration for %s project ...', framework)
     deployment_folder = join('_', 'deployment', technology)
+    if technology == 'python':
+        prune_cronjob_version(deployment_folder)
```

Comment:
I agree that it shouldn't be python dependent.  I just added this here because the entire application deployment tree is in a folder named "python" -- so the file/directory removal isn't relevant otherwise.  Making it just ignore the removal if files aren't present is probably the cleanest solution for now, but I agree it would perhaps be a good idea to make the kubernetes deployment less technology-dependent.

Reply:
Let's create a ticket for that. :+1: